packages/blocks/src/api/raw-handling/list-reducer.js

```diff
@@ -8,7 +8,9 @@ function isList( node ) {
 }
 
 function shallowTextContent( element ) {
-	return [ ...element.childNodes ]
+	// On mobile DOM implmentation NodeList is not spreadable so we need to convert it to an array.
```

Comment:
The comment doesn't make entirely sense to me. The web implementation also needs to be converted to a real array, but it happens to be spreadable. In general we should always use `Array.from`. I believe we do in most places. Is this the only instance where we spread to convert it to a real array?

Reply:
I removed the comment. I don't know if this is the only instance were we spread the NodeList, I just tracked down the bug I was seeing when doing paste without format on the RN app and got to this method.